README.md

```diff
@@ -109,6 +111,32 @@ class LoginCoordinator: Coordinator {
     
 To run the example project, clone the repo, and run ***pod install*** from the Example directory first.
 
+### Observable
+
+#### ViewModel
+Bindable property should be declared as Observable in ViewModel.
+```swift
+struct ExampleViewModel {
+    let name: Observable<String> = Observable<String>()

```

Comment:
Let's omit the unneeded property type: `let name = Observable<String>()`

Reply:
Fixed